lte/gateway/c/core/oai/tasks/amf/amf_app_handler.cpp

```diff
@@ -1111,34 +1113,45 @@ static int paging_t3513_handler(zloop_t* loop, int timer_id, void* arg) {
   MessageDef* message_p                          = nullptr;
   itti_ngap_paging_request_t* ngap_paging_notify = nullptr;
 
+  if (!amf_app_get_timer_arg(timer_id, &ue_id)) {
+    OAILOG_WARNING(
+        LOG_AMF_APP, "T3513: Invalid Timer Id expiration, Timer Id: %u\n",
+        timer_id);
+    OAILOG_FUNC_RETURN(LOG_NAS_AMF, RETURNok);
+  }
+
   ue_context = amf_ue_context_exists_amf_ue_ngap_id(ue_id);
 
-  if (!ue_context) {
-    OAILOG_ERROR(LOG_AMF_APP, "ue_context doest exist\n");
-    return -1;
+  if (ue_context == NULL) {
+    OAILOG_INFO(LOG_AMF_APP, "ue_context is NULL\n");
+    OAILOG_FUNC_RETURN(LOG_NAS_AMF, RETURNok);
   }
 
   // Get Paging Context
   amf_ctx    = &ue_context->amf_context;
   paging_ctx = &ue_context->paging_context;
 
   paging_ctx->m5_paging_response_timer.id = NAS5G_TIMER_INACTIVE_ID;
-  /*
-   * Increment the retransmission counter
-   */
-  paging_ctx->paging_retx_count += 1;
-  OAILOG_ERROR(
-      LOG_AMF_APP, "Timer: Incrementing retransmission_count to %d\n",
-      paging_ctx->paging_retx_count);
 
   if (paging_ctx->paging_retx_count < MAX_PAGING_RETRY_COUNT) {
+    paging_ctx->paging_retx_count += 1;
+
+    OAILOG_ERROR(
+        LOG_AMF_APP,
+        "T3513: timer has expired for ue id: %d with timer id: %d,"
+        "Sending Paging request again\n",
+        ue_id, timer_id);
+    /*
+     * Increment the retransmission counter
+     */
+    OAILOG_ERROR(
+        LOG_AMF_APP, "T3513: Incrementing retransmission_count to %d\n",
+        paging_ctx->paging_retx_count);
+
```

Comment:
I am wondering what is the convention to use ERROR or INFO, re-transmission is not an error at all.